prow/cmd/branchprotector/protect_test.go

```diff
@@ -1162,6 +1163,87 @@ branch-protection:
 				},
 			},
 		},
+		{
+			name:     "Global unmanaged: true makes us not do anything",
+			branches: []string{"cfgdef/repo1=master", "cfgdef/repo1=branch", "cfgdef/repo2=master"},
+			config: `
+branch-protection:
+  unmanaged: true
```

Comment:
Isn't this specific scenario rather pointless? Shouldn't we handle this by just dropping the cfgdef org?

Reply:
The test verifies that the global unmanaged setting makes us not do anything. If we have no config, we wouldn't do anything regardless, so we need to have config.

From a practical point of view this config is pointless and the new checkconfig check would complain about it, yes.